### ISSUE TITLE
[Módulo 03] [Aula 09] [Lógica de Programação] "Uncaught SyntaxError"

### ISSUE THREAD

**Issue by sofia-pellisoli** [cwi-reset/edicao-04-level-1#323]
Olá pessoal, 

Estou tentando realizar os exercícios da Aula 09 (Módulo 03) "Funções, procedimentos e comentários", mas sempre que eu rodo acontece um erro: "**"Uncaught SyntaxError: Identifier '_//aqui varia o nome//_' has already been declared"**"

... Pelo que entendi seria um erro se sintaxe, porque eu estaria declarando alguma coisa mais de uma vez, mas não sei se entendi certo e não consegui resolver... 

Alguém também teve este erro?
Se puderem me ajudar agradeço! :)

**Comment by superjhonatan** [comment]
bom dia @sofia-pellisoli , consegue postar uma print ou postar o código?

**Comment by sofia-pellisoli** [comment]
Claro! (na verdade cheguei no ponto de colar da resolução apresentada kkkkk)
![exercicio](https://user-images.githubusercontent.com/112525674/190913133-e686f626-993c-4d70-ab24-3ed8e767c25f.PNG)

**Comment by superjhonatan** [comment]
@sofia-pellisoli , não tem mais nada acima nem abaixo do trecho de código dessa print? Por que o erro apresentado é que foi declarado a variável mesmoNome duas vezes no mesmo scopo.

rodando seu código exatamente igual a print que você mandou esse erro não aconteceu.

![image](https://user-images.githubusercontent.com/112219528/190914219-620d7e95-78b0-4c9f-8975-0f5a99a6ee3e.png)

**Comment by sofia-pellisoli** [comment]
Não, nada acima e nada abaixo....
Mas agora reparei que estou rodando direto no Chrome e você no replit.com.... Vou tentar lá!

**Comment by superjhonatan** [comment]
no chrome da exatamente igual ao replit.com:

criando variavel `mesmoNome` antes de criar a função `mesmoNome`:
![image](https://user-images.githubusercontent.com/112219528/190914675-69e396c4-36f5-4154-a370-7401fc6d1a0c.png)

sem criar a variável `mesmoNome` antes de criar a função `mesmoNome`:
![image](https://user-images.githubusercontent.com/112219528/190914789-0a4abaa3-f489-4880-9578-a7578a6ee488.png)

**Comment by superjhonatan** [comment]
se o erro persistir no chrome, limpa o cache e tenta novamente

**Comment by sofia-pellisoli** [comment]
Certo, obrigada!!!

**Comment by sofia-pellisoli** [comment]
Funcionooooooou! Não acredito que era só isso, que vergonha HAHAHAHAHA
Muito obrigada por me ajudar em pleno domingo!

![exercicio](https://user-images.githubusercontent.com/112525674/190915635-482fabc5-4997-4c49-8a3d-9ba87ad1a10d.PNG)
